### PR TITLE
Algolia's DocSearch ready! ✨

### PR DESCRIPTION
**Pull request by LukyVj**
Hello guys! 

I'm happy to announce that this is the final PR for the DocSearch implementation on Tailwind.css! 
So, a few stuff to share with you, I added a few more files in order to have the best implem ever! 

But first, let see what happen here. 

I removed the "sidebar header" from the sidebar scope, because  _sidebar_ got a class that handle the scroll on the y axis if there is too much content. Therefore, the x axis is hidden, and because of that, the docsearch dropdown couldn't be visible. 

So I had to remove the header from the sidebar, even tho the look'n'feel is the same ( see here https://github.com/tailwindcss/tailwindcss/compare/master...LukyVj:master#diff-70b205842e0adc80d56e2ce469aadd47R8 ) 

To make sure this sidebar doesn't overlap the header, I had to create a new less file, called `sidebar.less` which only purpose is to offset the sidebar from top. Otherwise it would hide the header. 

I added also a file called `docsearch.less` which handle the color & responsiveness of the DocSearch dropdown. 

Talking about responsive, you'll notice 2 DocSearch declarations ( here : https://github.com/tailwindcss/tailwindcss/compare/master...LukyVj:master#diff-2802ff3e03db6b3656a5fc4e9b9467a5R30 ) 

And this is for reducing the amount of specific css that would have been required to adapt and reposition the search input on mobile. So this is an equivalent of having a [multiple input for docsearch](https://github.com/algolia/docsearch/issues/230), which is being implemented right now. 

So expect a PR really soon, that will remove the double DocSearch instance, and being replaced by 
`inputSelector: '#searchbox, #searchbox-mobile'`

I think that's pretty much it, feel free to ping me for anything you could need, or if you need me to change something. I'll be glad to do that for you! 

Lucas B & the Algolia team

### CODE REVIEW

**Comment by reinink** [comment]
@LukyVj Awesome, thanks so much! We'll get this fixed up and merged in. I really appreciate your help. 💪

File: docs/source/_layouts/master.blade.php [hunk 1]
```diff
@@ -18,12 +18,29 @@
...
+              indexName: 'tailwindcss',
+              inputSelector: '#searchbox'
+            });
+
+            // For mobile ( will be fixed soon, see : https://github.com/algolia/docsearch/issues/230 )
+            docsearch({
+              apiKey: '3df93446658cd9c4e314d4c02a052188',
+              indexName: 'tailwindcss',
+              inputSelector: '#searchbox-mobile',
+              debug: true
```
Comment (m1guelpf) [inline docs/source/_layouts/master.blade.php]: Why is debug enabled here?

Comment (LukyVj) [reply docs/source/_layouts/master.blade.php]: Good call! My bad, you can just remove it! Indeed it’s useful to when you need to inspect the dropdown. I forgot to remove it 😊

Comment (mdavis1982) [reply docs/source/_layouts/master.blade.php]: @Diddyy I think you tagged me by mistake here 😄

**Comment by fairbraxton** [comment]
This will also close #58!

**Comment by reinink** [comment]
This has been merged! 🎊 

One question @LukyVj: Is there anyway for us to view any analytics for DocSearch? Like, is there a control panel? I'm assuming no, but I figured I would ask.

Also, is there anyway to tweak the search settings? For example, right now if I search for "uppercase", I get no results. However, we definitely have "uppercase" in our docs: http://tailwindcss.dev/docs/text-style. Any suggestions would be much appreciated! ❤️ 

image

**Comment by LukyVj** [comment]
Hey @reinink 

> One question @LukyVj: Is there anyway for us to view any analytics for DocSearch? Like, is there a control panel? I'm assuming no, but I figured I would ask.

There is a way to have analytics! Let me reach out to you by email! 

And thanks for the support!!!!!!

**Comment by reinink** [comment]
@LukyVj AMAZING! ❤️ 

Also, searching for "uppercase" appears to be working now.

winning

**Comment by adamwathan** [comment]
Hey @LukyVj! I shipped a redesign of the docs the other day and am noticing now that I've broken the categories that used to appear in the left column of the search results 😬

image

Is there any way for me to fix this myself? Feel bad bugging you to do it for me!

**Comment by LukyVj** [comment]
Hey! Sure, I can take a look and debug it tomorow! 
But, seems like the tailwind website is broken / does not load for now!

**Comment by adamwathan** [comment]
Unfortunately a DNS outage messed up a ton of stuff for anyone using Hover for their domains :/ Apparently switching off of Google's DNS fixes it for most people; right now it's just a matter of waiting out the caching game.

**Comment by LukyVj** [comment]
So, I'm investigating, but seems like the content of the left column is just not here, I'm gonna find out why :)

**Comment by LukyVj** [comment]
Did you changed some DOM structure? The possible issue would be that the config we made for Tailwindcss at the time, is no longer working with a new website structure.

**Comment by adamwathan** [comment]
Yeah big time, I redesigned the whole site 😅Sorry! 😔

**Comment by LukyVj** [comment]
No worries! We are working on fixing it! I'll let you know ASAP :)

**Comment by LukyVj** [comment]
I'll let @s-pace fix this for you :)

**Comment by s-pace** [comment]
👋 ,

Done ✅, [you can have a look here](https://github.com/algolia/docsearch-configs/commit/381fe239d3c7fb8a71f613e6f14438f164bd01ba)

Already deployed and solved (check it, it's live)

Feel free to ping us at any time if you need help or if you have any feedback.

Cheers

**Comment by s-pace** [comment]
Btw, we do provide **analytics** 📊  on your DocSearch index 📝 
Please provide me the emails ✉️ of the collaborators 👥 to add to our analytics dashboard 😄

**Comment by adamwathan** [comment]
Amazing, thank you!

**Pull request merged**